firefly_client/firefly_client.py

```diff
@@ -1437,6 +1437,61 @@ def set_stretch_hprgb(self, plot_id, asinh_q_value=None, scaling_k=1.0,
         return_val['rv_lst'] = [d['rv'] for d in st_data]
         return return_val
 
+    def set_color(self, plot_id, image_type, **color_params):
```

Comment:
I don't think you should use `image_type`. It is nice for validation but the problem is that you have no way to know if `image_type` is what the plot ID actually is. The user could say `hips` and it is a three color.

The validation depends too much on what the user passes which might not the accurate. I think is is better to just document when parameters apply. 

An alternative would be to make two functions.

```python
# red,green,blue are all booleans
set_3c_used_bands(self, plot_id, red, green, blue)
```

and

```python
# bias and contrast are optional
set_color(self, plotId, colormap_id, bias, contrast)
```

Not that I think about it I think the two function approach is better. You also avoid `**color_params`

Reply:
Done!

Note: bias and contrast also apply for 3 color image, so your proposed function signatures and what I implemented looks bit different.